packages/angular/.eslintrc.js

```diff
@@ -39,7 +39,22 @@ module.exports = {
     },
     {
       extends: ['@aws-amplify/amplify-ui/jest'],
-      files: ['**/__mocks__/**', '**/__tests__/**'],
+      files: [
+        '**/__mocks__/**',
+        '**/__tests__/**',
+        './projects/ui-angular/src/lib/components/authenticator/components/setup-totp/setup-totp.component.spec.ts',
```

Comment:
Small comment explaining this would be good for future context

Reply:
Added